javascript/javascript1/week2/eventApplication.js

```diff
@@ -0,0 +1,18 @@
+function eventDay(daysToEvent){
+    const d= new Date();
+    const weekDay = new Array(7);
+    weekDay[0] = "Sunday";
+    weekDay[1] = "Monday";
+    weekDay[2] = "Tuesday";
+    weekDay[3] = "Wednesday";
+    weekDay[4] = "Thursday";
+    weekDay[5] = "Friday";
+    weekDay[6] = "Saturday";
```

Comment:
Definitely an interesting approach! 😄 

In the future, I would recommend you simply put the string directly in the array instead of using the `new Array` function.
```js
const days = ['Sunday','Monday', 'Tuesday', 'Wednesday', 'Thursday','Friday','Saturday']; 
```